pkg/internal/apis/config/encoding/testdata/v1alpha3/invalid-bad-indent.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+networking:
+  ipFamily: ipv6
+nodes:
+- role: control-plane
+- role: worker
+  extraMounts:
+  - containerPath: /foo
+    hostPath: /bar
+    readOnly: true
+    selinuxRelabel: false
+    propagation: Bidirectional
+  extraPortMappings:
+  - containerPort: 8080
+    hostPort: 8080
+    protocol: UDP
+  - role: worker
```

Comment:
we had an issue similar to this config, https://github.com/kubernetes-sigs/kind/issues/821

Reply:
this is now an error :-)